api-test/src/main/resources/injicertify/VCILandRegistryNeg/AuthorizationCodeForLandRegistry/AuthorizationCodeForLandRegistry.yml

```
@@ -19,10 +20,12 @@ AuthorizationCodeForLandRegistry:
 
    InjiCertify_ESignet_AuthorizationCode_ForLandRegistry_Cred_2_Pos_For_NegFlow_sid:
       endPoint: $ESIGNETMOCKIDABASEURL$/v1/esignet/authorization/auth-code
-      uniqueIdentifier: TC_esignetDependent_AuthorizationCode
+      uniqueIdentifier: TC_esignetDependent_AuthorizationCode_17
       description: Authorization Code for mock ida
       role: resident
       restMethod: post
+      additionalDependencies: TC_esignetDependent_AuthenticateUser_17
+      additionalDependencies: TC_esignetDependent_AuthenticateUser_17,TC_esignetDependent_AuthenticateUser_16,TC_esignetDependent_AuthenticateUser_18
```

YAMLlint (1.37.1)

[error] duplication of key "additionalDependencies" in mapping

(key-duplicates)